src/mscorlib/src/System/Threading/ThreadPool.cs

```diff
@@ -476,57 +513,66 @@ internal bool LocalFindAndPop(IThreadPoolWorkItem callback)
 
         public IThreadPoolWorkItem Dequeue(ThreadPoolWorkQueueThreadLocals tl, ref bool missedSteal)
         {
-            IThreadPoolWorkItem callback;
-            int wsqActiveObserved = WorkStealingQueueList.wsqActive;
-            if (wsqActiveObserved > 0)
+            IThreadPoolWorkItem callback = null;
+            try
```

Comment:
I added try/finally blocks.  After the try all the change is whitespace up until the finally block.